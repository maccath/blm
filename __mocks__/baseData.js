const baseData = {
  site: {
    siteMetadata: {
      author: "Site Author",
      description: "Site description",
      title: "Site Title",
    },
  },
  placeholderImage: {
    childImageSharp: {
      fluid: {
        src: "icon.png",
        srcSet: "icon.png 50w, icon.png 100w",
        sizes: "(max-width: 200px) 100vw, 200px",
        base64: "data:image/png;base64,iVBORw0KG",
        aspectRatio: 1,
      },
    },
  },
}

export default baseData
