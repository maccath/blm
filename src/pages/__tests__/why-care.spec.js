import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import "jest-styled-components"
import baseData from "../../../__mocks__/baseData"

import WhyCare from "../why-care"

beforeEach(() => {
  useStaticQuery.mockImplementation(() => baseData)
})

describe("Why Care? page", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<WhyCare />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
