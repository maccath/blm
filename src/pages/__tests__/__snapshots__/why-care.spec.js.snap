// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Why Care? page renders correctly 1`] = `
<div
  style={
    Object {
      "opacity": 1,
      "transition": "opacity 500ms ease-in-out",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "0 1.0875rem 1.45rem",
      }
    }
  >
    <main>
      <h1>
        Why should I care?
      </h1>
      <p>
        Although the resources I've posted so far reference racism in the USA - it
is not a problem exclusive to America. The white supremacist culture
exists in the UK too. We can see this in the following ways:
      </p>
      <ul>
        <li>
          Black males were 3.9 times more likely to die from COVID19 than the White
group. Among females, death rates were 3.3 times higher in the Black ethnic
group than the White group - according to 
          <a
            href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/889195/disparities_review.pdf"
          >
            Public Health England review
          </a>
        </li>
        <li>
          There were 4 stop and searches for every 1,000 White people, compared with
38 for every 1,000 Black people - according to 
          <a
            href="https://www.ethnicity-facts-figures.service.gov.uk/crime-justice-and-the-law/policing/stop-and-search/latest#by-ethnicity"
          >
            Government statistics
          </a>
        </li>
        <li>
          Black people were most likely to be detained under the Mental Health Act
in the year to March 2019, with 306.8 detentions per 100,000 people. White
people had the lowest rate of detention â€“ 72.9 detentions per 100,000 people -
according to 
          <a
            href="https://www.ethnicity-facts-figures.service.gov.uk/health/mental-health/detentions-under-the-mental-health-act/latest#by-ethnicity-5-ethnic-groups"
          >
            Government statistics
          </a>
        </li>
      </ul>
      <p>
        In reference to the current issues facing America, the 
        <a
          href="https://www.amnesty.org.uk/press-releases/usa-protests-uk-should-review-exports-security-equipment-us-police-forces"
        >
          UK government could be unwittingly supporting police violence by exporting riot gear including rubber bullets
        </a>
        .
Such export of items likely to be used for internal repression is against UK
policy. The UK has 
        <a
          href="https://www.independent.co.uk/news/world/americas/america-is-running-out-of-lethal-injection-drugs-because-of-a-european-embargo-to-end-the-death-10106933.html"
        >
          refused to supply chemicals for execution of prisoners in the USA which has derailed capital punishment in the US
        </a>
        .
      </p>
      <p>
        <strong>
          Our policies and actions can and do make a difference around the world.
        </strong>
      </p>
      <nav
        className="css-1b6p5hc"
      >
        <a
          className="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
          href="/who-cares"
        >
          Why should I take it from you?
        </a>
        <a
          className="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
          href="/what-to-do"
        >
          That's pretty damning.
        </a>
      </nav>
    </main>
    <footer
      className="flex justify-center space-x-5 css-1bxtzlt"
    >
      <div>
        <a
          href="https://thenounproject.com/cesarreynoso562/"
        >
          Fist icon by Cesar Reynoso
        </a>
      </div>
      <div>
        <a
          href="https://github.com/maccath/blm"
        >
          <svg
            className="inline mr-2"
            fill="currentColor"
            height={15}
            viewBox="0 0 24 24"
            width={15}
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              GitHub
            </title>
            <path
              d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"
            />
          </svg>
          Contribute on GitHub
        </a>
      </div>
    </footer>
  </div>
</div>
`;
